### PR TITLE
Dependencies versions update

### PR DESCRIPTION
**Pull request by osuleymanova**
Gradle version updated
compileSdk and targetSdk versions updated
gradle. properties added
libraries number versions updated
jcenter() replaced with mavenCentral()

### CODE REVIEW

File: build.gradle [hunk 1]
```diff
@@ -124,16 +124,16 @@ publishing {
 
 dependencies {
     implementation 'com.google.protobuf:protobuf-lite:3.0.1'
-    implementation 'androidx.core:core:1.0.0'
-    implementation 'com.google.android.material:material:1.2.0-rc01'
+    implementation 'androidx.core:core:1.7.0'
+    implementation 'com.google.android.material:material:1.6.0-alpha02'
```
Comment (calren) [inline build.gradle:128]: i think we should stick with the latest stable version instead of using an alpha version

File: build.gradle [hunk 2]
```diff
@@ -13,19 +13,19 @@ buildscript {
...
     }
 }
 
 apply plugin: 'com.android.library'
 
 android {
-    compileSdkVersion 29
-    buildToolsVersion '29.0.3'
+    compileSdkVersion 31
+    buildToolsVersion '30.0.3'
```
Comment (calren) [inline build.gradle:25]: we can just take out the buildToolsVersion now  https://stackoverflow.com/questions/24521017/android-gradle-buildtoolsversion-vs-compilesdkversion

File: build.gradle (hunk 1 above)
Comment (osuleymanova) [reply build.gradle]: Thank you Caren, the version changed to 1.5.0.

File: build.gradle (hunk 2 above)
Comment (osuleymanova) [reply build.gradle]: buildToolsVersion '30.0.3' line removed. Thank you!

**Review by calren — APPROVED** [review]

**Comment by calren** [comment]
I mistakenly thought this was an update to a codelab, instead of an update to the actual test framework. Android DevRel doesn't own this repository, so I'm not sure we should be the ones to update it.

**Pull request closed without merging**